koku/cost_models/cost_model_manager.py

```diff
@@ -84,8 +84,8 @@ def update_provider_uuids(self, provider_uuids):
                 raise CostModelException(log_msg)
             CostModelMap.objects.create(cost_model=self._model, provider_uuid=provider_uuid)
 
-        start_date = DateHelper().this_month_start
-        end_date = DateHelper().today
+        start_date = str(DateHelper().this_month_start.date())
+        end_date = str(DateHelper().today.date())
```

Comment:
While casting to string may work, I think using `.strftime()` is more readable and ensures the string formatting doesn't change unexpectedly.

Reply:
👍  Good idea!